labman/gui/templates/normalization.html

```diff
@@ -276,7 +276,8 @@ <h3>Normalization</h3>
 </div>
 
 <!-- Reformat -->
-<div class="form-check" style='padding-bottom:10px'>
+<!-- display set to none (to hide from interface) until reformatting is fixed -->
```

Comment:
is there an issue for this? worth adding here?

Reply:
Yes, we have an issue for this already; it is #323 'Fix "reformat" option in Normalize gDNA Plates bug'.